tests/test_logger.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""Watchmaker main test module."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+import os
+import shutil
+import sys
+
+from watchmaker.logger import (LOG_LEVELS, exception_hook, make_log_dir,
+                               prepare_logging)
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
```

Comment:
Doesn't seem to support it.  `six` doesn't seem to have a module rename for unittest and the support for assertions is only supported for Python 2.7+, so we would be stuck in the same boat for Python 2.6.  :(